Doc/library/subprocess.rst

```diff
@@ -516,8 +516,24 @@ functions.
 
    If given, *startupinfo* will be a :class:`STARTUPINFO` object, which is
    passed to the underlying ``CreateProcess`` function.
-   *creationflags*, if given, can be :data:`CREATE_NEW_CONSOLE` or
-   :data:`CREATE_NEW_PROCESS_GROUP`. (Windows only)
+   *creationflags*, if given, can be
```

Comment:
This change seems unrelated and I preferred the previous formatting.

Reply:
i'm not sure how this is unrelated? this shows where the added constants should be used?

Reply:
Ah, when I wrote my comment, the doc was formatted differently.

Ok, now I suggest to format these constants as a list:

```
* a
* b
* c
* ...
```

(correctly indented for the current section)

Try to build the documentation to see how it's rendered in HTML. On Unix, it's "make -C Doc html". I don't know how to build it on Windows :-(

Reply:
unfortunately i don't currently have access to a unix box (windows 10 broke my dual boot) and currently on a work pc so don't even have git client, doing this through the web interface... so i can't build the docs to see how they look

Reply:
>  *creationflags*, if given, can be

I checked in MSDN: in fact, you can specify *multiple* flags at once:
https://msdn.microsoft.com/en-us/library/windows/desktop/ms686331(v=vs.85).aspx

dwFlags: "A bitfield that determines whether certain STARTUPINFO members are used when the process creates a window. **This member can be one or more of the following values.**"

---

By the way, please add a column after "can be" ("can be:"), and an empty line after.

I propose: "can be one or more of the following flags:".